catalogue/webapp/test/utils/dates.test.ts

```diff
@@ -182,54 +212,63 @@ describe('filterExceptionalClosedDates', () => {
     );
 
     expect(result).toEqual([
-      london('2019-12-03'),
-      london('2019-12-05'),
-      london('2019-12-17'),
-      london('2019-12-31'),
-      london('2020-01-07'),
-      london('2020-01-09'),
-      london('2020-01-18'),
+      new Date('2019-12-03'),
+      new Date('2019-12-05'),
+      new Date('2019-12-17'),
+      new Date('2019-12-31'),
+      new Date('2020-01-07'),
+      new Date('2020-01-09'),
+      new Date('2020-01-18'),
     ]);
   });
 });
 
 describe('includedRegularClosedDays', () => {
   it('determines how many regular closed days occur between the start and end dates, inclusive', () => {
     const result = includedRegularClosedDays({
-      startDate: london('2020-01-03'),
-      endDate: london('2020-01-16'),
+      startDate: new Date('2020-01-03'),
+      endDate: new Date('2020-01-16'),
       closedDays: [0, 1, 4],
     });
 
+    // This is the date range:
+    //
+    //        January 2020
+    //     Su Mo Tu We Th Fr Sa
+    //                     3  4
+    //      5  6  7  8  9 10 11
+    //     12 13 14 15 16
```

Comment:
This comes from the Unix command `cal jan 2020`, which is super nifty for this sort of thing.